### PR TITLE
feat: uo

### PR DESCRIPTION
**Pull request by kawaemon**

### CODE REVIEW

File: src/bot/uo/mod.rs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+        use Command::*;
+
+        let msg = match parsed.command {
+            Status => {
+                let prob = self.prob_percent.lock().await;
+                format!("```{UO}確率: {prob}%```")
+            }
+            Reroll => {
+                const KAWAEMON_DISCORD_USER_ID: u64 = 391857452360007680;
```
Comment (su8ru) [inline src/bot/uo/mod.rs]: ｳｰｫ

**Review by su8ru — CHANGES_REQUESTED** [review]

File: src/bot/uo/mod.rs (hunk 1 above)
Comment (kawaemon) [reply src/bot/uo/mod.rs]: [ｳｰｫ](https://github.com/approvers/rusty-ponyo/pull/101/commits/2289dce2a348ad9e869821c64ae988510ac7a414)

**Review by 2RiniaR — APPROVED** [review]
ｳｰｫ

**Review by su8ru — APPROVED** [review]
ｵｰｩ

**Pull request merged**